src/layouts/ReviewRequest/components/Comments/SendCommentForm.tsx

```diff
@@ -44,7 +44,12 @@ export const SendCommentForm = ({
   const queryClient = useQueryClient()
 
   const handleUpdateNotifications = async ({ comment }: CommentFormProps) => {
-    await updateNotifications({ siteName, requestId, message: comment })
+    try {
+      await updateNotifications({ siteName, requestId, message: comment })
```

Comment:
The `updateComments` hook is a mutation hook, would it be better in this case if we leveraged on react-query and update this section to use their `onSuccess` and `onError` callback functions?

Reply:
Doesn't work unfortunately! `onError` still results in a thrown error, which messes with the new implementation of react-hook-form - the field straight up won't work anymore until it's in a no longer submitting state. Even if we use the error states to determine whether the button is useable, the user can't go back to change the field because the "unhandled error" is blocking further changes to the fields. So the only solution is to have these extra try catch blocks which are just there to consume the error

Reply:
Wait actually you're right - i thought it was being blocked by rhf - it's actually blocked by `isReadOnly`. Removed `isSubmitting` from the comments form! The others i've opted to leave in because they require prop drilling if we want to make use of `isLoading` from react-query instead